search-service/src/test/kotlin/com/egm/stellio/search/web/TemporalEntityHandlerTests.kt

```diff
@@ -371,26 +372,45 @@ class TemporalEntityHandlerTests {
 
     @Test
     fun `it should return a single entity if the entity has two temporal properties`() {
-        val entityTemporalProperty1 = TemporalEntityAttribute(
```

Comment:
I've the feeling that this test is getting big and multi-purpose, isn't it?

Reply:
In fact I didn't get much the purpose of this test, I can leave it untouched and create another one if it 's ok

Reply:
Indeed, I think it is a kind of legacy test that lost most of its purpose...